sources/mep-kilogram-definition-en.adoc

```diff
@@ -290,7 +292,7 @@ Neither the realizations of the metre nor the second have been affected by the R
 
 === Electrical units
 
-The ampere was previously defined in terms of the second, the metre and the kilogram, and by giving a fixed numerical value to the magnetic constant stem:[mu_0], whose unit is stem:[rm(kg)" "rm(m)" "rm(s)^{-2} " "rm(A)^{-2}] (equivalently, stem:[rm(N)" "rm(A)^{-2}] or stem:[rm(H)" "rm(m)^{-1}]). The ampere is now defined in terms of the second and a fixed numerical value for the elementary charge stem:[e], whose unit is stem:[rm(A)" "rm(s)]. The fact that the Planck constant now has a defined numerical value is of great utility to electrical metrology, as described in the _mise en pratique_ for the ampere <<ampere>>.
+The ampere was previously defined in terms of the second, the metre and the kilogram, and by giving a fixed numerical value to the magnetic constant stem:[ii(mu)_0], whose unit is stem:[rm(kg)" "rm(m)" "rm(s)^{-2} " "rm(A)^{-2}] (equivalently, stem:[rm(N)" "rm(A)^{-2}] or stem:[rm(H)" "rm(m)^{-1}]). The ampere is now defined in terms of the second and a fixed numerical value for the elementary charge stem:[e], whose unit is stem:[rm(A)" "rm(s)]. The fact that the Planck constant now has a defined numerical value is of great utility to electrical metrology, as described in the _mise en pratique_ for the ampere <<ampere>>.
```

Comment:
Throughout the document, Greek letters are being italicised, with the exception of Δ, π. It is possible that they are exceptions because they are used mathematically, and the rest are italicised because they are variables. If that is the case, the markup needs to be preserved, to be resolved in UnitsML later; if there is a different pattern at work, I need to know about it.

Reply:
I believe Δ was not italicized in any of these documents, but I'm truly not sure about π. @manuel489 , what do you think?

Reply:
In my view, both non-italic and italic `pi` are used according to certain case.
It seems like italic `pi` is used when an equal (or inequality) sign is present (i.e. equations). And where is not, upright `pi` is used.
This is a capture for fr-brochure:

![fr-brochure_pi](https://user-images.githubusercontent.com/33627611/102676451-f7d66700-4173-11eb-8f00-0240ae69e1f5.PNG)

Reply:
Although, this is not always true. *mep-ampere-definition-en* uses upright `pi` in any case.

Reply:
But in `mep-candela-si-app2`, there is:

![candela-italic](https://user-images.githubusercontent.com/33627611/102676650-f6596e80-4174-11eb-8633-11cf57b00f47.PNG)

And there also is:

![candela-not-italic](https://user-images.githubusercontent.com/33627611/102676662-04a78a80-4175-11eb-83f5-746fa1a4d666.PNG)

(Both from the same document)

Reply:
I feel that this is due to practice inconsistency instead of semantic reasons. Let me clarify with BIPM.

Reply:
Sent clarification to BIPM, pending reply.

Reply:
This is clarified in metanorma/metanorma-bipm#93.